lib/Platform.cs

```diff
@@ -177,6 +177,7 @@ static Platform()
             if (Name == PlatformName.Posix)
             {
                 // check again if this might be Mac OS X
+				// TODO: isn't there a better program to read "Mac OS X" ?
```

Comment:
"Better" in what sense? What is wrong about sw_vers?

Reply:
It's just MacOSX, not UNIX or POSIX. What about `uname` or some Environment variable you see using `export` ?

Reply:
I do not have a Mac

Reply:
Using uname, we could find out that we are running on Darwin, which is not necessarily Mac OS X. But maybe that is really the info we need.